sdk/core/core-amqp/samples/receive.ts

```diff
@@ -1,23 +1,61 @@
-// Copyright (c) Microsoft Corporation. All rights reserved.
-// Licensed under the MIT License. See License.txt in the project root for license information.
+/*
+  Copyright (c) Microsoft Corporation. All rights reserved.
+  Licensed under the MIT Licence.
+
+   This sample demonstrates how to receive messages/events from Service Bus/Event Hubs.
+*/
 
-import * as dotenv from "dotenv";
-dotenv.config(); // Optional for loading environment configuration from a .env (config) file
 import {
   Receiver,
   ReceiverOptions,
-  EventContext,
   ReceiverEvents,
   delay,
-  types
+  types,
+  EventContext
 } from "rhea-promise";
-import { authenticate, connectionContext, connectionConfig, path } from "./cbsAuth";
+
+import {
+  ConnectionContextBase,
+  CreateConnectionContextBaseParameters,
+  CbsResponse,
+  ConnectionConfig,
+  TokenType,
+  SharedKeyCredential
+} from "@azure/core-amqp";
+
+// Define connection string and related entity path here
+const connectionString = "";
+const path = "";
+
+const connectionConfig = ConnectionConfig.create(connectionString, path);
+const parameters: CreateConnectionContextBaseParameters = {
+  config: connectionConfig,
+  connectionProperties: {
+    product: "MSJSClient",
+    userAgent: "/js-core-amqp",
+    version: "0.1.0"
+  }
+};
+const connectionContext = ConnectionContextBase.create(parameters);
+
+async function authenticate(audience: string): Promise<CbsResponse> {
+  await connectionContext.cbsSession.init();
+  const sharedTokenCredential = <SharedKeyCredential>connectionContext.tokenCredential;
+  const tokenObject = sharedTokenCredential.getToken(audience);
+  const result = await connectionContext.cbsSession.negotiateClaim(
+    audience,
+    tokenObject,
+    TokenType.CbsTokenTypeSas
+  );
+  console.log("Result is: %O", result);
+  return result;
+}
 
 async function main(): Promise<void> {
-  await authenticate(`${connectionConfig.endpoint}${path}`, false);
+  await authenticate(`${connectionConfig.endpoint}${connectionConfig.entityPath}`);
```

Comment:
Have we tested this? Shouldnt we be passing the consumer group and partition infor here?

Reply:
Yes, it works fine, no need to pass consumer group and partition information here.